internal/pkg/client/oras/oras.go

```diff
@@ -86,7 +86,7 @@ func newOrasUploadTransport() http.RoundTripper {
 	}
 }
 
-func getResolver(ctx context.Context, ociAuth *ocitypes.DockerAuthConfig, noHTTPS, push bool) (remotes.Resolver, error) {
+func getResolver(ctx context.Context, ociAuth *ocitypes.DockerAuthConfig, noHTTPS, push, orasResolver bool) (remotes.Resolver, error) {
```

Comment:
I don't think the name `orasResolver` is very descriptive.  How about something like `progressBar` or `includeProgressBar`.

Reply:
renamed to `progressBar` thanks!
https://github.com/apptainer/apptainer/pull/1587/commits/aab871d7c4c217c169908a6e6afddc9f49654c3f